src/backend/schemas/file.py

```diff
@@ -10,7 +10,8 @@ class File(BaseModel):
     updated_at: datetime.datetime
 
     user_id: str
-    conversation_id: str
+    conversation_id: Optional[str] = None
+    file_content: Optional[str] = None
```

Comment:
This schema is used for both the request but also the type to return for methods in the file service (since if we use Compass we can't map it to the DB file model schema).